packages/trader/src/Modules/MT5/Components/mt5-poa.jsx

```diff
@@ -300,27 +277,21 @@ class MT5POA extends Component {
-                                                <div className='mt5-proof-of-address__file-area'>
-                                                    <div className='mt5-proof-of-address__description'>
-                                                        <Description />
-                                                    </div>
-                                                    <div className='mt5-proof-of-address__file-upload'>
-                                                        <FileUploaderContainer
-                                                            onRef={ref => this.setFileUploadRef(ref)}
-                                                            is_description_disabled
-                                                            onFileDrop={({ document_file: df, file_error_message }) =>
-                                                                this.onFileDrop(
-                                                                    df,
-                                                                    file_error_message,
-                                                                    setFieldTouched,
-                                                                    setFieldValue
-                                                                )
-                                                            }
-                                                        />
-                                                        {errors.document_file && touched.document_file && (
-                                                            <p className='dc-field-error'>{errors.document_file}</p>
-                                                        )}
-                                                    </div>
+                                                <div className='mt5-proof-of-address__file-upload'>
+                                                    <FileUploaderContainer
+                                                        onRef={ref => this.setFileUploadRef(ref)}
+                                                        onFileDrop={({ document_file: df, file_error_message }) =>
+                                                            this.onFileDrop(
+                                                                df,
+                                                                file_error_message,
+                                                                setFieldTouched,
+                                                                setFieldValue
+                                                            )
+                                                        }
+                                                    />
+                                                    {errors.document_file && touched.document_file && (
+                                                        <p className='dc-field-error'>{errors.document_file}</p>
```

Comment:
Why is a `dc-` class being used outside of Components? Just curious

Reply:
I wanted to have this paragraph have the same error styling as the default one we have in the components, so I just reused that class.